src/test/java/service/impl/ActivityParserServiceImplTest.java

```diff
@@ -0,0 +1,46 @@
+package service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+import model.Activity;
+import org.junit.Test;
+import service.ActivityParserService;
+
+public class ActivityParserServiceImplTest {
+    private static final ActivityParserService PARSER_SERVICE = new ActivityParserServiceImpl();
+    private static final String[] EMPTY_STRING = {};
+    private static final String[] DEFAULT_STRING = {
+            "b,banana,20",
+            "b,apple,100",
+            "s,banana,100",
+            "p,banana,13",
+            "r,apple,10",
+            "p,apple,20",
+            "p,banana,5",
+            "s,banana,50",
+            "b,kiwi,10"
+    };
+    private static List<Activity> activityList;
```

Comment:
no need to include data type in the variable's name, Java is a strongly typed language